frontend/src/less/components/searchbox.less

```diff
@@ -1,15 +1,44 @@
 #searchbox {
+    width: @width-searchbox;
+    max-width: 100%;
+    margin: 0 auto;
+    position: relative;
+    @media (max-width: @screen-sm-min) {
+        margin-top: 30px;
+    }
+    @media (max-width: @screen-xs-max) {
+        padding: 0 20px
+        width: 100%;
+    }
+
     span {
         padding-right: 20px;
         line-height: 52px;
     }
 
+    .how {
+        position: absolute;
+        top: 15px;
+        left: -90px;
+        padding-right: 0px;
+        line-height: normal;
+        @media (max-width: @screen-sm-min) {
```

Comment:
@amcatko should have been @screen-xs-max :) (using max-width, you probably want xx-xx-max and vice versa)